assemblyline/common/identify.py

```diff
@@ -371,6 +378,7 @@
      r'BinHex|InstallShield CAB|Transport Neutral Encapsulation Format|archive data|compress|mcrypt'
      r'|MS Windows HtmlHelp Data|current ar archive|cpio archive|ISO 9660'],
     ['meta', '^MS Windows shortcut'],
+    ['metadata', 'MS Windows Vista Event Log'],
```

Comment:
I think for the sake of consistency, 
```
['meta', '^MS Windows shortcut'], -> ['meta', r'^MS Windows shortcut']
['metadata', 'MS Windows Vista Event Log'], -> ['metadata', r'MS Windows Vista Event Log']
```
Also, for the record I think it's gross that we are using nested lists where a list of dictionaries could suffice.